daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/GlobalKey.scala

```diff
@@ -0,0 +1,39 @@
+package com.daml.lf
```

Comment:
You are going to need to add the licence header here!

Reply:
fixed.